[INFO ] 2022-05-19 15:53:55.470 [Test worker] c.s.c.DepartmentController - In getAllTeachers (id = [4])
[INFO ] 2022-05-19 15:53:55.509 [Test worker] c.s.s.i.DepartmentServiceImpl - Enter into getAllTeachers with department id 4
[INFO ] 2022-05-19 15:53:55.510 [Test worker] c.s.r.i.DepartmentRepositoryImpl - In getAllTeachers (departmentId = [4])
[INFO ] 2022-05-19 15:53:56.460 [Test worker] c.s.c.DepartmentController - In getDisabled ()
[INFO ] 2022-05-19 15:53:56.462 [Test worker] c.s.s.i.DepartmentServiceImpl - Enter into getAll of getDisabled
[INFO ] 2022-05-19 15:53:56.462 [Test worker] c.s.r.i.BasicRepositoryImpl - In getDisabled
[INFO ] 2022-05-19 15:53:57.132 [Test worker] c.s.c.DepartmentController - In save (departmentDTO = [com.softserve.dto.DepartmentDTO@57540])
[INFO ] 2022-05-19 15:53:57.133 [Test worker] c.s.s.i.DepartmentServiceImpl - In save(entity = [Department(id=null, name=null, disable=false)]
[INFO ] 2022-05-19 15:53:57.140 [Test worker] c.s.r.i.DepartmentRepositoryImpl - In isExistsByName(name = [null])
[INFO ] 2022-05-19 15:53:57.140 [Test worker] c.s.r.i.BasicRepositoryImpl - In save(entity = [Department(id=null, name=null, disable=false)]
[ERROR] 2022-05-19 15:53:57.186 [Test worker] c.s.e.h.GlobalExceptionHandler - Validation failed for classes [com.softserve.entity.Department] during persist time for groups [javax.validation.groups.Default, ]
List of constraint violations:[
	ConstraintViolationImpl{interpolatedMessage='Name cannot be blank', propertyPath=name, rootBeanClass=class com.softserve.entity.Department, messageTemplate='Name cannot be blank'}
]
[INFO ] 2022-05-19 15:53:57.741 [Test worker] c.s.c.DepartmentController - In save (departmentDTO = [com.softserve.dto.DepartmentDTO@56b57])
[INFO ] 2022-05-19 15:53:57.742 [Test worker] c.s.s.i.DepartmentServiceImpl - In save(entity = [Department(id=null, name=, disable=false)]
[INFO ] 2022-05-19 15:53:57.742 [Test worker] c.s.r.i.DepartmentRepositoryImpl - In isExistsByName(name = [])
[INFO ] 2022-05-19 15:53:57.840 [Test worker] c.s.r.i.BasicRepositoryImpl - In save(entity = [Department(id=null, name=, disable=false)]
[ERROR] 2022-05-19 15:53:57.933 [Test worker] c.s.e.h.GlobalExceptionHandler - Validation failed for classes [com.softserve.entity.Department] during persist time for groups [javax.validation.groups.Default, ]
List of constraint violations:[
	ConstraintViolationImpl{interpolatedMessage='Name cannot be blank', propertyPath=name, rootBeanClass=class com.softserve.entity.Department, messageTemplate='Name cannot be blank'}
]
[INFO ] 2022-05-19 15:53:58.343 [Test worker] c.s.c.DepartmentController - In save (departmentDTO = [com.softserve.dto.DepartmentDTO@65757])
[INFO ] 2022-05-19 15:53:58.344 [Test worker] c.s.s.i.DepartmentServiceImpl - In save(entity = [Department(id=null, name=  , disable=false)]
[INFO ] 2022-05-19 15:53:58.345 [Test worker] c.s.r.i.DepartmentRepositoryImpl - In isExistsByName(name = [  ])
[INFO ] 2022-05-19 15:53:58.436 [Test worker] c.s.r.i.BasicRepositoryImpl - In save(entity = [Department(id=null, name=  , disable=false)]
[ERROR] 2022-05-19 15:53:58.530 [Test worker] c.s.e.h.GlobalExceptionHandler - Validation failed for classes [com.softserve.entity.Department] during persist time for groups [javax.validation.groups.Default, ]
List of constraint violations:[
	ConstraintViolationImpl{interpolatedMessage='Name cannot be blank', propertyPath=name, rootBeanClass=class com.softserve.entity.Department, messageTemplate='Name cannot be blank'}
]
[INFO ] 2022-05-19 15:53:58.946 [Test worker] c.s.c.DepartmentController - In getAll ()
[INFO ] 2022-05-19 15:53:58.947 [Test worker] c.s.s.i.DepartmentServiceImpl - In getAll()
[INFO ] 2022-05-19 15:53:58.947 [Test worker] c.s.r.i.DepartmentRepositoryImpl - In getAll()
[INFO ] 2022-05-19 15:53:59.531 [Test worker] c.s.c.DepartmentController - In getById (id = [20])
[INFO ] 2022-05-19 15:53:59.533 [Test worker] c.s.s.i.DepartmentServiceImpl - In getById(id = [20])
[INFO ] 2022-05-19 15:53:59.533 [Test worker] c.s.r.i.BasicRepositoryImpl - In findById(id = [20])
[ERROR] 2022-05-19 15:53:59.739 [Test worker] c.s.e.h.GlobalExceptionHandler - Department was not found for parameters {id=20}
[INFO ] 2022-05-19 15:54:00.136 [Test worker] c.s.c.DepartmentController - In save (departmentDTO = [com.softserve.dto.DepartmentDTO@cba1c9a1])
[INFO ] 2022-05-19 15:54:00.137 [Test worker] c.s.s.i.DepartmentServiceImpl - In save(entity = [Department(id=null, name=save new departments, disable=false)]
[INFO ] 2022-05-19 15:54:00.137 [Test worker] c.s.r.i.DepartmentRepositoryImpl - In isExistsByName(name = [save new departments])
[INFO ] 2022-05-19 15:54:00.229 [Test worker] c.s.r.i.BasicRepositoryImpl - In save(entity = [Department(id=null, name=save new departments, disable=false)]
[INFO ] 2022-05-19 15:54:00.874 [Test worker] c.s.c.DepartmentController - In update (departmentDTO = [com.softserve.dto.DepartmentDTO@1bc40e7e])
[INFO ] 2022-05-19 15:54:00.875 [Test worker] c.s.s.i.DepartmentServiceImpl - In update(entity = [Department(id=4, name=Department4, disable=false)]
[INFO ] 2022-05-19 15:54:00.875 [Test worker] c.s.r.i.DepartmentRepositoryImpl - In isExistsByTitleIgnoringId(id = [4], name = [Department4])
[INFO ] 2022-05-19 15:54:00.970 [Test worker] c.s.r.i.BasicRepositoryImpl - In update(entity = [Department(id=4, name=Department4, disable=false)]
[INFO ] 2022-05-19 15:54:01.558 [Test worker] c.s.c.DepartmentController - In getById (id = [4])
[INFO ] 2022-05-19 15:54:01.559 [Test worker] c.s.s.i.DepartmentServiceImpl - In getById(id = [4])
[INFO ] 2022-05-19 15:54:01.559 [Test worker] c.s.r.i.BasicRepositoryImpl - In findById(id = [4])
[INFO ] 2022-05-19 15:54:02.139 [Test worker] c.s.c.DepartmentController - In save (departmentDTO = [com.softserve.dto.DepartmentDTO@1bc620cd])
[INFO ] 2022-05-19 15:54:02.140 [Test worker] c.s.s.i.DepartmentServiceImpl - In save(entity = [Department(id=null, name=Department4, disable=false)]
[INFO ] 2022-05-19 15:54:02.140 [Test worker] c.s.r.i.DepartmentRepositoryImpl - In isExistsByName(name = [Department4])
[ERROR] 2022-05-19 15:54:02.330 [Test worker] c.s.e.h.GlobalExceptionHandler - Department with parameter {name=Department4} already exists
